pkgs/development/interpreters/php/generic.nix

```diff
@@ -141,7 +141,7 @@ let
           phpWithExtensions = symlinkJoin {
             name = "php-with-extensions-${version}";
             inherit (php) version;
-            nativeBuildInputs = [ makeWrapper ];
+            nativeBuildInputs = [ makeBinaryWrapper ];
```

Comment:
Is there behavioural difference with a binary wrapper? If not perhaps split it into a separate commit with its own rationale.

Reply:
No, there shouldn't be a difference for the particular case. Using the binary wrapper allows using it in a shebang on darwin because execvp doesn't like shell scripts. Do you want me to split the commits?

Reply:
Yeah, since it is unrelated to this change, a separate commit with explanation would be better.

Reply:
Reverted that change so that this PR can get it, will open a 2nd issue once this has landed.